EIPS/eip-3779.md

```diff
@@ -132,9 +132,9 @@ For simplicity's sake we assume a few helper functions.
    * dynamic jumps match a valid (_jumpsrc_, _jumpdest_) in the `vjumptable` or `vtraptable`
    * all jump destinations are `JUMPDEST` bytes and not in immediate data.
 * `remove_items()` returns the number of items removed from the `stack` by an instruction
-* `add_items() returns the number of items added to the `stack`.  Items are added as 0xFFFFFFFF.
+* `add_items()` returns the number of items added to the `stack`.  Items are added as `0xFFFFFFFF`.
    The `PC`, `PUSH…`, `SWAP…`,  `DUP…`, `JUMP`, and `JUMPI` instructions are handled separately.
-```
+```go
```

Comment:
This actually needs an empty new line to work.

Reply:
Syntax highlighting works for me at https://github.com/gumb0/EIPs/blob/adca9bfa5593a797edc72d182c224214a1a2b0a1/EIPS/eip-3779.md

Reply:
It doesn't work on eips.ethereum.org.  Github and Jekyll use different markdown engines -- they should be identical, but Github's differs a tiny bit.